### ISSUE TITLE
[Adaptive Cards - Restaurant]: Talkback is announcing twice role information as Link and Button for the 'More Info' link.*

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#492]
### Target Platforms

Android

### SDK Version

1.0.1

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: Android 16
Device: Samsung S24
Screen Reader: Talkback
Adaptive Card Android App version: 1.0.1

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'Restaurant'.
8. Navigate to 'More Info' link and Observe the issue.

**Actual Result:**
While navigating to the 'Restaurant' card, Talkback is announcing twice role information as 'Link and button'. It is announcing as 'More info link button.'

**Expected Result:**
Talkback should announce correct role information as link for the More Info. It should announce as 'More Info link.'

**User Impact:**
It will impact the talkback user as they will get confused if the screen reader will announce the twice role for any control also due to this user will not able whether on activating the control what action will be performed.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/1ebbf94f-4f72-4f79-88c7-bfa6d83981a8

### Card JSON

```JSON
NA
```
